OWNERS_ALIASES

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#owners_aliases
+
+aliases:
+  update-approvers:
+  - LalatenduMohanty
+  - jottofar
+  - vrutkovs
+  - wking
+  update-reviewers:
```

Comment:
shouldn't these names be more specific/clearly point to the team/component?  "update" is pretty vague/likely to be interpreted in other ways

Reply:
Moved to a `cluster-version-operator-test-case-` prefix in 27d2c285ed -> 802d3898b8.